src/requestforattestation/RequestForAttestation.ts

```diff
@@ -164,6 +207,16 @@ export default class RequestForAttestation implements IRequestForAttestation {
     return this.verifySignature()
   }
 
+  /**
+   * Verifies the signature inside the [[RequestForAttestation]] object, an unsuccessful signature will stop the Tx (Transaction).
+   *
+   * @returns Whether the verification of the claimers signature was successful.
+   * @example
+   * ```javascript
+   * RequestForAttestation.verifySignature();
```

Comment:
Better, one thing though: `verifySignature` is not static, so it needs to be called on an instance aka a `requestForAttestation` (the capitalized version is reserved to designate the class).
Same for other instance methods: `verifyData`... etc.